yb-voyager/cmd/common.go

```diff
@@ -1178,7 +1178,7 @@ type AssessmentIssueYugabyteD struct {
 	MinimumVersionsFixedIn map[string]*ybversion.YBVersion `json:"MinimumVersionsFixedIn"` // key: series (2024.1, 2.21, etc)
 
 	// Store Type-specific details - extensible, can refer any struct
-	Details json.RawMessage `json:"Details,omitempty"`
+	Details map[string]interface{} `json:"Details,omitempty"`
```

Comment:
Just thinking if changing the datatype from `json.RawMessage` to `map[string]interface{}` can cause any runtime issue in parsing at YBD side.

Can you test this by writing some sample code?